src/test/ui/typeck/issue-87181-empty-tuple-fields.stderr

```diff
@@ -0,0 +1,19 @@
+error[E0599]: no method named `foo` found for fn item `fn() -> Inner {Inner}` in the current scope
+  --> src/test/ui/typeck/issue-87181-empty-tuple-fields.rs:16:15
```

Comment:
I think you need to run `./x.py test --bless src/test/ui/typeck/issue-87181-empty-tuple-fields.rs`.

Reply:
I think I might be missing something. I ran your command and it generated the stderr file, but the test still fails, and it's showing me some JSON.